source/common/grpc/common.cc

```diff
@@ -24,6 +24,20 @@ namespace Grpc {
 
 const std::string Common::GRPC_CONTENT_TYPE{"application/grpc"};
 
+Status::GrpcStatus Common::getGrpcStatus(const Http::HeaderMap& trailers) {
```

Comment:
Could have `Common::validateTrailers()` make use of this to do its gRPC status parsing.